### PR TITLE
Dashboard icons update

### PR DESCRIPTION
**Pull request by shutterspeak**
Updated icons for Quick Tags, ReplyViewer, and TagViewer on the
dashboard

### CODE REVIEW

**Comment by nightpool** [comment]
Can you add before and after screenshots? Thanks!

File: Extensions/replyviewer.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ XKit.extensions.replyviewer = new Object({
...
+	button_icon:
+	"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAABGdBTUEAALGPC/xhBQAAAsVJREFUOBHNlL9LW1EUx+9L4w+aLCVtrSkpQQRpIzYgDkKpdA7BLIlbqF2VaDbrH1DdoiEZutSSTtUOimQubaEOLqkaS4uUUMEgGOiQgM3Pfs4z75GgFocOPXDeOffc7/nee8899yn1j0W7jC+ZTNorlcrTRqMxDsaDOtACmtU0baOjo+P11NRUkXGbXEi4vLwcgiiO9pC8R8YW9oTxTfxR7CDjYzQyMzOz2sp4jjAWiz0H8AL9QcLk7Ozsx9YE8ZeWlh5DuoLbh85Ho9EFiYu0EcrO6vX6W+KrDofjWTgcLumoCz6pVMpWKBReMRWyWCwTxk5NwmbNDli5BNnQ38gM/ibpDiexUdN+qanVmGxegNRszCCTRarV6jy7niB+l8UOsG+cTmcsFAqVBcfxJ4l/kHy4EhaDkOA44D2jZgBHINthfs7lcrm9Xm9Xb2/vfXCLR0dH24lEQm5eCV7yJF/GJiG+h+BnCQoZ5r3dbr8XDAY1iCzZbFZBbAkEAqq7u/tBrVb7FI/HHwoe2UL1BVoJHRRX+kxhVyC3+f3+a5CpTCajyuWybt1ut/L5fFbKcANMSvDs8AQjfdq2wwKA2xIEPAmotLm5Wcvn84rjqs7OTt3mcjmVTqerzP9Cw4InT/rzbDMSaMp37ID41GUb86RYLP5cW1trQFr3eDzq8PCwvr6+rk5PT/etVuujSCTyRfDIKJoVxzwyx9xnrNdBJoSUpCHcRYhyHPs3xF/Z1Ry3PDI9Pa0TUMcxdigvZ0PyzLbhmCWCXRI0pPlW5xmLnhOjuck7lrctAJOQ4B3GxxK8ihhkYPvInTB+FCYhEz3olQjlLfPszLdsPDvZSBshtdAALxAfxh9mZQ0rzf0NV1rjFtr6tzHfMHFdTEISrxPpxw6QvIv/Dm2gg2gAlb4TUvkfvrzsf2gScqN+LsZGI+/KOyXx/5A/n/9bBKmk/qYAAAAASUVORK5CYII=",
```
Comment (nightpool) [inline Extensions/replyviewer.js]: remove the newline here

File: Extensions/tagviewer.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ XKit.extensions.tagviewer = new Object({
...
+	button_icon:
+	"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAABGdBTUEAALGPC/xhBQAAAr5JREFUOBGtlN9rkmEUx5/XqWtOmFMkQUf+eC+SQQipyGbE2C6MoAuvuhCE2NUGOf+BKKp7Fbzo0m6DSAi6MFbgj1pzVwtkgsxfQ200Rq9EYu3te6RXnGy6QQcO53me95zPOc/znOflotHofVEUQ1AZg3AQsv/kN+xntVr9bHV19UhaHGVlBIODDZzvpBgfSopvAsZr7XY7G4/HDRiPFS4SiXyB1+HGxsbds7yxAy/A7/CtrlAoltbX15tn+UlrvW1KE8CfQwWy0looFMqg8juYm7rd7odxlZ4CIughqlGTlYBkLwMdBsZQTRuM2CDwMtCxZzgMHnemVOEfbFM+HHjefNz2CfgJ27xNmc+DDK+PgsqoaRGwT63xP6C9V0GtQC0BsIlahCpAAh/mD6CLUB20Ad2EvoAP9W5PBs90amrqZv+ZSVCVSjUXCAS+Tk5O6re3tw8KhQKPl6LVaDRNh8OxPz8/bwfwPWhrsD+QmAP0Dew9uVx+ow+kdHt7e16TybRZLBa5TCYjIkBmNBontFota7VaYqPREJFI8Pv9u3q9np7iSiwWe3RyckI7eRIOhx9PEIgEGXidTvd2a2urmMvl5qxWqwyBMqVSSTBmt9s5r9fLAarMZrPXAN6dnZ19iiI8nU6nByNOr0LArmCc39nZOUqn07cQyJxOJzs+PmaJRIKS0V+IBYNBhq0zABmOg+EImMvl+jY9PW3B958EvHD/kfOwoMoKYPRH6kv/DGnLWP2Isyvn8/lFnufF5eVlrlqtslKpxGw2GzMYDCyVSon1ep1zu93lhYUFAi2hujINSPpAmgB6HSZVq9VKyWTSA0f54KU0m02Gm+R8Pl/FYrF04bsCnwrFSnIKSIuAXoV5iZvjpbYRBEE3MzPDzGaz0uPxtHFRSfj02oZiLiQA+6CvoAfQX0hQBvg1xu5RgL8jX3FUaeAzggAAAABJRU5ErkJggg==",
```
Comment (nightpool) [inline Extensions/tagviewer.js]: ibid

File: Extensions/xkit_patches.js [hunk 3]
```diff
@@ -1670,7 +1670,8 @@ XKit.tools.getParameterByName = function(name){
 					XKit.tools.add_css("." + class_name + ":after {" +
 							" background-image: url('" + icon + "') !important;" +
 							" background-size: auto auto !important;" +
-							" margin-top: -9px !important; " +
+							" margin-top: -7px !important; " +
+							" opacity: 1 !important; " +
```
Comment (nightpool) [inline Extensions/xkit_patches.js]: you should modify the opacities in the xkit_patches css file instead of here.

Comment (shutterspeak) [inline Extensions/xkit_patches.js]: I've modified every line with an opacity value, but  the only way I could get it to work is by adding that line in the js file. I'll keep working on it!

**Comment by shutterspeak** [comment]
icons_old
Old Icons
icons_new
New icons

Again, I think the resolution is messing up the colors, because I've triple-checked that the colors match what they look like on Tumblr:
icons_color

**Comment by nightpool** [comment]
@shutterspeak my only concern about the icons is that we're losing the symmetry/consistency between the TagViewer and ReplyViewer icons. Also, it doesn't actually view "replies" in the sense that it's viewing the replies on post, "ReplyViewer" means that it shows reblogs with content. (Probably needs a rename, but that's a different discussion).

So using the tumblr "reply" bubble here is probably the wrong user interface metaphor

**Comment by shutterspeak** [comment]
I can turn it back into the original conversation bubble, although it might be confusing to have two close together, especially considering that only of them is for actual replies (with the other being reblog content). I don't actually use the extension, but I have it installed so I'll go see if I can think of a better icon and replace it.

**Comment by nightpool** [comment]
@shutterspeak there are pros and cons to both interfaces. (ours will load more replies, and in a more compact format) I would prefer to keep ReplyViewer around for now, even though much of it's functionality is achievable natively.

**Comment by shutterspeak** [comment]
Ah, I see! I can make it the reblog icon and keep the eye (since I guess on some level you are viewing who reblogged it along with what they say) if that's a better metaphorical fit.

**Comment by nightpool** [comment]
@shutterspeak that's a great idea.

**Comment by shutterspeak** [comment]
icons_new

@nightpool thoughts?

File: Extensions/xkit_patches.js (hunk 3 above)
Comment (hobinjk) [inline Extensions/xkit_patches.js]: @nightpool  `opacity: 1 !important` occurs a ton of other places in the file, I'm inclined to allow this modification as it is locally consistent

**Comment by nightpool** [comment]
I'm on my phone, but if that was my only complaint, then +1

**Comment by hobinjk** [comment]
The quick tags button is a bit too see-through. The line in xkit_patches that defaults the opacity to 0.22 should be deleted or changed to 1.0. It's [at this line](https://github.com/new-xkit/XKit/blob/master/Extensions/xkit_patches.css#L87)

**Pull request closed without merging**